tests/test_rest.py

```diff
@@ -43,19 +44,34 @@ def test_records_rest(app, es, loaded_example_licenses):
             resp = client.get('/licenses/_suggest?text=mit')
             resp_json = json.loads(resp.get_data(as_text=True))
             assert resp.status_code == 200
-
             options = resp_json['text'][0]['options']
-            assert len(options) == 2
-            assert {(o['payload']['id'], o['text']) for o in options} == {
-                ('MIT', 'MIT License'),
-                ('mitre', 'MITRE Collaborative Virtual Workspace License '
-                 '(CVW License)')}
-
+            if ES_VERSION[0] == 2:
+                assert len(options) == 2
+                assert {(o['payload']['id'], o['text']) for o in options} == {
```

Comment:
Actually, `invenio-records-rest` makes sure `_source` is always there with these keys (i.e. the same case as below for `ES_VERSION[0] > 2`). I think we can keep the `_source` version of the tests (since ES2 will be removed soon)